packages/superset-ui-chart-controls/src/operators/boxplotOperator.ts

```diff
@@ -54,7 +49,7 @@ export const boxplotOperator: PostProcessingFactory<PostProcessingBoxplot | unde
       options: {
         whisker_type: whiskerType,
         percentiles,
-        groupby: ensureIsArray(queryObject.groupby as QueryFormColumn[]),
+        groupby: ensureIsArray(groupby),
```

Comment:
`groupby` is deprecated on `QueryObject` and is ambigous with `columns`, hence we should use `groupby` from `formData` here, as it carries a specific meaning for the Boxplot chart.